versions/next/schema.json

```diff
@@ -742,7 +742,16 @@
               "type": "string"
             },
             "headers": {
-              "$ref": "#/definitions/schema"
+              "type": "object",
+              "additionalProperties": false,
+              "patternProperties": {
```

Comment:
You can just use `additionalProperties: { oneOf...` instead of patternProperties.

Reply:
OK I'll change